usermenu.php

```diff
@@ -169,6 +169,28 @@ function usermenu_civicrm_navigationMenu(&$menu) {
   _usermenu_civix_navigationMenu($menu);
 } // */
 
+/**
+ * Implements hook_civicrm_alterContent().
+ * Adds extra settings fields to the Civicase Admin Settings form.
+ *
+ * @link https://docs.civicrm.org/dev/en/latest/hooks/hook_civicrm_alterContent/
+ */
+function usermenu_civicrm_alterContent (&$content, $context, $templateName, $form) {
+  $isViewingMiscSettingsForm = get_class($form) === CRM_Admin_Form_Setting_Miscellaneous::class;
+
+  if (!$isViewingMiscSettingsForm) {
+    return;
+  }
+
+  $settingsTemplate = &CRM_Core_Smarty::singleton();
+  $settingsTemplateHtml = $settingsTemplate->fetchWith('CRM/Usermenu/Admin/Form/Settings.tpl', []);
+
+  $doc = phpQuery::newDocumentHTML($content);
+  $doc->find('table.form-layout:eq(1) tr:last')->append($settingsTemplateHtml);
+
+  $content = $doc->getDocument();
```

Comment:
Code is good for now, but if we keep adding new logic in this hook, will be a good time to move the code to separate classes.

Reply:
Yes, I know we should use the hook pattern as in other extensions. I'll keep this in mind as soon as we touch this file again.